Entries/copy-X/copy-X.md

```diff
@@ -168,36 +186,187 @@ No samples currently known, and thus no information either.
+	* Music Cleaning Lab 2004 Deluxe[^MX_MCL2004D_1][^MX_MCL2004D_2]
+	* Music Maker[^MX_MMD] (Dutch?)
+	* Music Maker 3.0[^MX_AR]
+	* Music Maker 5[^MX_MM5]
+	* Music Maker 7 Deluxe[^MX_M3MG_MM7D_MVD]
+	* Music Maker 10 Deluxe[^MX_AR]
+	* Music Maker 2003 Deluxe[^MX_FaCD2_MM2003D_MS2003D]
+	* Music Maker Generation 6[^MX_MMG6] (Swedish?)
+	* Music Studio 7 Deluxe[^MX_MM7D]
+	* Music Studio 2003 Deluxe[^MX_FaCD2_MM2003D_MS2003D]
+	* Music Studio 2005 Deluxe[^MX_MS2005D] (Dutch?)
+	* Music Studio V2000[^MX_MSV2K_PaKW2E_TMM_ACL3]
+	* Piano & Keyboard Workshop 2nd Editon[^MX_MSV2K_PaKW2E_TMM_ACL3]
+	* Retten Sie Ihre Videokassetten[^MX_FaD7_RSIV]
+	* Techno Music Maker[^MX_MSV2K_PaKW2E_TMM_ACL3][^MX_TMM] (1.0?)
+	* Video Deluxe[^MX_M3MG_MM7D_MVD]
+
+At some point Xara was folded into Magix. Due to the similarity of various different product and company names, it's difficult to determine if Xara was already using copy-X before this, or if Xara only began using copy-X after being folded into Magix.
+
+* Xara/Magix:
+	* XaraXtreme 4.0[^XA_XX4_Link1][^XA_XX4_Link2]
+	* Xtreme Photostory on CD & DVD 6[^XA_XPCD6]
+
+Due to the frankly absurd number of releases from Magix that seem to have copy-X, which seem to be available even in different regional releases, it can most likely be assumed that any Magix product in the 2000s probably has copy-X.
+
+* Kiddinx:
+	* Playmobil - Gefangen in der Drachenfestung[^RR_FuFS_RudWmF_aR_dP_CuV_PGidD]
+	* Pettson Och Findus Födelsedagskatten[^KDDX_POFF]
+	* Słoń Benjamin - Ruch Uliczny[^KDDX_SBRU] 
+	* Bibi und Tina - Pferdestarke Spielesammlung[^KDDX_BuTPS]
+
+Likely a fair bit more, but it's strangely hard to find information. 
+
+* Terzio:
+	* Das Traumfresserchen 
+	* Löwenzahn 5[^TZ_L57FLHWWP][^TZ_L5]
+	* Löwenzahn 7[^TZ_L57FLHWWP][^TZ_L7]
+	* Löwenzahn 8[^TZ_L8]
+	* Flugzeuge bauen und Häuser bauen mit Willy Werkel[^TZ_L57FLHWWP]
+	* Fritz & Fertig - Schach lernen und trainieren[^TZ_FFSCLuT]
+	* Fritz & Fertig Terzio Schach lernen und trainieren CD-ROM ab 8[^TZ_FFTSlutCD8] 
+	* Willy Werkel: Häuser bauen (Rerelease)[^TZ_WWHhR]
+
+Likely many others also exist. Of note, Terzio appears to have began switching to ProtectDISC[^TZ_WWPD] starting around ~2008, similar to 16 Tons Entertainment.
+
+* Tivola:
+
+While many unconfirmed samples could be listed here, there's virtually no point in doing so.
+
+* Radon Labs / Cornelsen Verlag: // TODO: Incorporate companies into confirmed?
+
+* One-offs (so far):
+
+	* Deluxe Ski Jump 3[^DSJ_DLs]
+	* Deluxe Ski Jump 4[^DSJ_DLs]
+		
+	* Ronja Räubertochter[^RR_FuFS_RudWmF_aR_dP_CuV_PGidD]
+		
+	* Rund um die Welt mit Felix[^RR_FuFS_RudWmF_aR_dP_CuV_PGidD]
+
+	* RTL Skispringen 2001[^RTL_DE]
+	* Skoki Narciarskie 2001[^RTL_PL_AR] - Polski Zwycięzca (Polish RTL Skispringen 2001?)
+
+Rest of the RTL Skispringen series in both regions appears to switch protectCD after 2001.
+		
+[^Genius_Physik]: [Genius: Unternehmen Physik](https://archive.org/details/DRML-copy-X-samples)
```

Comment:
Please go over these citations for archived date, none here seem to include them.

Reply:
Should be fixed now.